### PR TITLE
fix: Rename Values to Configure

### PR DESCRIPTION
**Pull request by shivani170**
# Description
  In Helm App detail > Rename Values tab with a gear icon to Configure

Screenshot 2022-11-17 at 5 42 16 PM

Fixes  [AB#1203](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/Devtron%20Team/Issues/?workitem=1203)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
1- Go to the Helm Chart
2- Go to second tab

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/headers/ChartHeader.component.tsx [hunk 1]
```diff
@@ -102,17 +103,18 @@ function ChartHeaderComponent() {
...
+                        className="tab-list__tab-link flex"
                         onClick={(event) => {
                             ReactGA.event({
                                 category: 'App',
                                 action: 'Values Clicked',
                             })
                         }}
                     >
-                        Values
+                        <Settings className={`tab-list__icon icon-dim-16 fcn-7 mr-4`} />
```
Comment (vivek-devtron) [inline src/components/v2/headers/ChartHeader.component.tsx:116]: We can pass the value directly as string. no need to interpolate

File: src/components/v2/headers/ChartHeader.component.tsx [hunk 2]
```diff
@@ -9,20 +9,21 @@ import { get } from '../../../services/api'
...
-    const { path } = useRouteMatch()
+    const { path, url } = useRouteMatch()
     const appDetails = IndexStore.getAppDetails()
 
     function handleBreadcrumbChartChange(selected) {
         const newUrl = generatePath(path, { appId: selected.installedAppId, envId: selected.environmentId })
         history.push(newUrl)
     }
-
+console.log(match)
```
Comment (vivek-devtron) [inline src/components/v2/headers/ChartHeader.component.tsx:26]: please remove console.log

File: src/components/v2/headers/ChartHeader.component.tsx [hunk 3]
```diff
@@ -9,20 +9,21 @@ import { get } from '../../../services/api'
 import { handleUTCTime } from '../../common'
 import './header.scss'
 import IndexStore from '../appDetails/index.store'
+import { ReactComponent as Settings } from '../../../assets/icons/ic-settings.svg'
 import PageHeader from '../../common/header/PageHeader'
-
+import '../../app/details/app.css'
```
Comment (vivek-devtron) [inline src/components/v2/headers/ChartHeader.component.tsx:14]: Why are we adding app.css here?

File: src/components/v2/headers/ChartHeader.component.tsx (hunk 2 above)
Comment (shivani170) [reply src/components/v2/headers/ChartHeader.component.tsx]: Done

File: src/components/v2/headers/ChartHeader.component.tsx (hunk 1 above)
Comment (shivani170) [reply src/components/v2/headers/ChartHeader.component.tsx]: Done

**Review by vivek-devtron — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**